packages/fxa-payments-server/.storybook/components/MockApp.tsx

```diff
@@ -7,21 +7,25 @@ type MockAppProps = {
   children: ReactNode
 };
 
-export const MockApp = ({ children }: MockAppProps) => <>
-  <div id="fox-logo"></div>
-  <div id="stage" style={{ opacity: 1 }}>
-    <div id="main-content" className="card visible">
-      <div className="app">
-        <StripeProvider apiKey={config.STRIPE_API_KEY}>
-          <MockLoader>
-            {children}
-          </MockLoader>
-        </StripeProvider>
-      </div>
-    </div>
-  </div>
-  <a id="about-mozilla" rel="author" target="_blank"
-    href="https://www.mozilla.org/about/?utm_source=firefox-accounts&amp;utm_medium=Referral"></a>
-</>;
+export const MockApp = ({
+  children
+}: MockAppProps) => {
+  // HACK: Set attributes on <html> dynamically because it's very hard to
+  // customize the template in Storybook
+  useEffect(() => {
+    const el = document.documentElement;
+    el.setAttribute('lang', 'en');
+    el.setAttribute('dir', 'ltr');
+    el.setAttribute('class', 'js no-touch no-reveal-pw getusermedia');
+  }, []);
+
+  return (
+    <StripeProvider apiKey={config.STRIPE_API_KEY}>
```

Comment:
Note: this will have an unidentified conflict with #1280

Reply:
I think that's okay for now. I'm planning to work on styling the payment form next, which uses this token. So I can include an update for this there